src/common/routes.jsx

```diff
@@ -50,6 +51,10 @@ export const routes = {
         path: '/palmid',
         component: Palmid,
     },
+    palmida: {
+        path: '/palmida',
+        component: Palmid_advanced,
+    },
     access: {
```

Comment:
this is to test out new component I built, if this looks good we can use it with /palmid